pac4j-core/src/main/java/org/pac4j/core/exception/HttpAction.java

```diff
@@ -49,28 +49,30 @@ public static HttpAction redirect(final String message, final WebContext context
     }
 
     /**
-     * Build an HTTP Ok without any content.
+     * Build an HTTP Ok.
      *
      * @param message message
      * @param context context
+     * @param content content
      * @return an HTTP ok
      */
-    public static HttpAction ok(final String message, final WebContext context) {
-        return ok(message, context, "");
+    public static HttpAction ok(final String message, final WebContext context, final String content) {
+        context.setResponseStatus(HttpConstants.OK);
+        context.writeResponseContent(content);
+        return new HttpAction(message, HttpConstants.OK);
     }
 
     /**
-     * Build an HTTP Ok.
+     * Build an HTTP No content.
      *
      * @param message message
      * @param context context
-     * @param content content
-     * @return an HTTP ok
+     * @return an HTTP No content
      */
-    public static HttpAction ok(final String message, final WebContext context, String content) {
-        context.setResponseStatus(HttpConstants.OK);
-        context.writeResponseContent(content);
-        return new HttpAction(message, HttpConstants.OK);
+    public static HttpAction noContent(final String message, final WebContext context) {
```

Comment:
I'm wondering, do we really need this `message`? Same for `ok()` actually...

Reply:
You're right, I'm not sure this is really useful. I should do some experiments...

I opened: https://github.com/pac4j/pac4j/issues/1004 to test the change.